KioskTests/Bid Fulfillment/RegistrationMobileViewControllerTests.swift

```diff
@@ -19,5 +19,20 @@ class RegistrationMobileViewControllerTests: QuickSpec {
             subject.bidDetails.newUser.phoneNumber = "1234567890"
             expect(subject).to( haveValidSnapshot() )
         }
+
+        it("unbinds bidDetails on viewWillDisappear:") {
+            let runLifecycleOfViewController = { (bidDetails: BidDetails) -> RegistrationMobileViewController in
+                let subject = RegistrationMobileViewController.instantiateFromStoryboard(fulfillmentStoryboard)
+                subject.bidDetails = bidDetails
+                subject.loadViewProgrammatically()
+                subject.viewWillDisappear(false)
```

Comment:
It doesn't really matter to me – the test is that the subscriptions end when `viewWillDisappear:` is called, so that's what I called. Happy to change.